### PR TITLE
[ZH] Add `Rules/Explicit Content` and several new translations in `Beatmap`

### PR DESCRIPTION
**Pull request by Sunnyboy971**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Keep up with #8708

### CODE REVIEW

File: wiki/Beatmap/Countdown/zh.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+*关于锦标赛计时器，参见：[锦标赛管理命令](/wiki/osu!tourney/Tournament_management_commands)*\
+*关于倒计时间的规定，参见：[谱面上架标准 (RC) § 常规 § 准则](/wiki/Ranking_Criteria#guidelines)*\
+*另见： [Offset （消歧义）](/wiki/Disambiguation/Offset)*
+
+在一张[谱面](/wiki/Beatmap)开头，可能会出现**倒计时**。这对于突然迅速开始的歌曲很有用，但大多数谱面不需要启用它。除了 [osu!taiko](/wiki/Game_mode/osu!taiko) 谱面，可以在[song setup](/wiki/Client/Beatmap_editor/Song_Setup)菜单的`Design`标签页下开启或关闭倒计时。
+
+当倒计时启用时，可以使用 `Countdown Speed` 按钮 (`Half`, `Normal`, `Double`) 与 `Countdown Offset` 按钮控制倒计时动画。 `Countdown Speed` 按钮将倒计时动画调节为 0.5 倍速、原速与 2 倍速。 `Countdown Offset` 设置在第一个[打击物件](/wiki/Hit_object)前[节奏](/wiki/Beatmapping/Beats_per_minute)上的补偿。例如，一个带有补偿 2 的倒计时动画会比正常情况早 2 拍出现。对于有开头的图，跳过开头会直接到达倒计时。
+
+倒计时动画可以通过[皮肤](/wiki/Skinning/Interface#countdown)修改。
```
Comment (YumeMuzi) [inline wiki/Beatmap/Countdown/zh.md:17]:
```suggestion
倒计时动画可以通过[自定义皮肤](/wiki/Skinning/Interface#countdown)修改。
```

File: wiki/Beatmap/Countdown/zh.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - 计时器
+---
+
+# 倒计时 (countdown)
+
+*关于锦标赛计时器，参见：[锦标赛管理命令](/wiki/osu!tourney/Tournament_management_commands)*\
+*关于倒计时间的规定，参见：[谱面上架标准 (RC) § 常规 § 准则](/wiki/Ranking_Criteria#guidelines)*\
+*另见： [Offset （消歧义）](/wiki/Disambiguation/Offset)*
+
+在一张[谱面](/wiki/Beatmap)开头，可能会出现**倒计时**。这对于突然迅速开始的歌曲很有用，但大多数谱面不需要启用它。除了 [osu!taiko](/wiki/Game_mode/osu!taiko) 谱面，可以在[song setup](/wiki/Client/Beatmap_editor/Song_Setup)菜单的`Design`标签页下开启或关闭倒计时。
```
Comment (YumeMuzi) [inline wiki/Beatmap/Countdown/zh.md:13]:
```suggestion
在一张[谱面](/wiki/Beatmap)开头，可能会出现**倒计时**。这对于突然迅速开始的歌曲很有用，但除了 [osu!taiko](/wiki/Game_mode/osu!taiko) 谱面之外，不需要在大多数谱面中使用它。可以在[歌曲设置 (Song Setup)](/wiki/Client/Beatmap_editor/Song_Setup)菜单的 `设计 (Design)` 标签页下开启或关闭倒计时。
```

File: wiki/Rules/Explicit_Content/zh.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# 少儿不宜内容
```
Comment (YumeMuzi) [inline wiki/Rules/Explicit_Content/zh.md:1]: 推荐将 `少儿不宜` 改为 `不良内容`。（按中文语境来说这游戏大多数玩家都是未成年吧，，）

File: wiki/Rules/Explicit_Content/zh.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+只有持续、显著、极具煽动性或高度重复地使用脏话，才能算作少儿不宜。
+
+此外，旗帜鲜明、持续讨论有争议的话题，或者任何理性的人认为“大量”的内容，都应该被认为少儿不宜。其中一些例子如下:
+
+- 自杀的意象、影响或后果
+- 大量的暴力暗示
+- 描述或讨论暴力后果时提及血腥细节
+- 明显提及性行为，或者暗示性行为
+
+如果想要辨别某些内容是否应该被标记为少儿不宜，你可以看看主流音频流媒体服务是否在其平台上将其列为少儿不宜。访问[Spotify](https://www.spotify.com)是一个很好的开始。
```
Comment (YumeMuzi) [inline wiki/Rules/Explicit_Content/zh.md:28]:
```suggestion
如果想要辨别某些内容是否应该被标记为少儿不宜，你可以看看主流音频流媒体服务是否在其平台上将其列为少儿不宜。访问 [Spotify](https://www.spotify.com) 是一个很好的开始。
```
加空格

File: wiki/Beatmap/Beatmap_description/zh.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# 谱面说明
+
+**铺面说明**是[谱面](/wiki/Beatmap)页下的一个小区域，使用 [BBCode](/wiki/BBCode) 向玩家提供关于此谱面的信息。它通常强调谱面制作者认为足够重要的一些事实，或者用来宣传其他制谱项目。
+
+## 使用与信息
+
+[上架 (ranked)](/wiki/Beatmap/Category#ranked)谱面的流行说明通常包含以下要素：
+
+- 所有 [Guest 难度](/wiki/Beatmap/Guest_difficulty)及其制作者的列表
```
Comment (YumeMuzi) [inline wiki/Beatmap/Beatmap_description/zh.md:9]:
```suggestion
- 所有[客串难度](/wiki/Beatmap/Guest_difficulty)及其制作者的列表
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
seems fine.
但这些文章的其他段落可能也需要修改对应的用词（谱面介绍、少儿不宜等）。需要注意。

File: wiki/Rules/Explicit_Content/zh.md (hunk 3 above)
Comment (Sunnyboy971) [reply wiki/Rules/Explicit_Content/zh.md:1]: 我一开始翻译这篇文章的时候是按照 Lazer 版上的说法来的。确实比较奇怪...;)

**Review by YumeMuzi — APPROVED** [review]

**Review by YumeMuzi — APPROVED** [review]

**Comment by YumeMuzi** [comment]
only [osu! wiki maintainers]'(https://osu.ppy.sh/wiki/en/People/osu%21_wiki_maintainers) approved could make the PR merged, be careful!

**Comment by Sunnyboy971** [comment]
Maybe a native reviewer's approval is needed for them to ensure the translation's quality...
So I'm used to request reviews after commits :smile_cat:

**Comment by TicClick** [comment]
an explicit approval shows that someone has actually read through the new changes, so thank you. for really minor changes it is not necessary, but anything else kind of requires a new look.

**Review by TicClick — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**